packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -36,6 +36,29 @@ export class DatabaseClusterEngine {
     this.singleUserRotationApplication = singleUserRotationApplication;
     this.multiUserRotationApplication = multiUserRotationApplication;
   }
+
+  /**
+   * Get this engine's default parameter group family for given version
+   */
+  public getClusterParameterGroupFamily(engineVersion?: string): string {
```

Comment:
1. Rename this to `defaultParameterGroupFamily`

2. Is there a link that you can add to the documentation which describes what the default parameter group family is? (I'm unfamiliar with the concept and I'd like to myself learn what a parameter group family is)

3. Is this *the* parameter group family, or only the default?

Reply:
I suppose it's  _the_ parameter group, at least at this point in time there seems to be only one family per engine version. I have renamed it as such.